src/js/providers/html5.js

```diff
@@ -636,12 +640,27 @@ define([
                 _this.once(events.JWPLAYER_MEDIA_SEEKED, _this.play);
                 return;
             }
+            _beforeResumeHandler();
             _play();
         };
 
         this.pause = function() {
             clearTimeout(_playbackTimeout);
             _videotag.pause();
+            _beforeResumeHandler = function() {
+                var unpausing = _videotag.paused && _videotag.currentTime;
+                if (unpausing && _videotag.duration === Infinity) {
+                    var end = _getSeekableEnd();
+                    var seekableDuration = end - _getSeekableStart();
+                    var isLiveNotDvr = seekableDuration < MIN_DVR_DURATION;
+                    var behindLiveEdge = end - _videotag.currentTime;
+                    if (isLiveNotDvr && end && behindLiveEdge > 15) {
```

Comment:
Safari's seekable range may reset to  0 - n, where n < `_videotag.currentTime`, which would cause `behindLiveEdge` to be negative. This stalls the player until the seekable range catches up to the `_videotag.currentTime` value. We should also resume from the edge of the live stream if `behindLiveEdge < 0`.